Source/Scene/Scene.js

```diff
@@ -690,10 +702,17 @@ define([
 
         var camera = new Camera(this);
         this._camera = camera;
-        this._cameraClone = Camera.clone(camera);
         this._screenSpaceCameraController = new ScreenSpaceCameraController(this);
         this._mapMode2D = defaultValue(options.mapMode2D, MapMode2D.INFINITE_SCROLL);
 
+        if (this._logDepthBuffer) {
+            this._camera.frustum.near = 0.1;
+            this._camera.frustum.far = 10000000000.0;
```

Comment:
Should this depend on `logarithmicDepthFarToNearRatio`?

Reply:
It doesn't need to. The default camera far plane doesn't depend to `farToNearRatio`.